tensorflow/contrib/fused_conv/kernels/fused_conv2d_bias_activation_op.cc

```diff
@@ -525,7 +525,7 @@ void LaunchFusedConv2DBiasActivationOp<GPUDevice, T, BiasType, ScaleType>::
   static int64 ConvolveScratchSize = GetDnnWorkspaceLimit(
       // default value is in bytes despite the name of the environment variable
       "TF_CUDNN_WORKSPACE_LIMIT_IN_MB", 1LL << 32  // 4GB
-  );
+      );
```

Comment:
The indentation looks different from my clang-format output. Which clang-format version do you have? I have 8.0.

Reply:
It's also different from mine. I just followed what's shown in `Experimental clang-format Check` test to edit the indentation.

Reply:
Maybe something is mis-configured. Oh well.